### PR TITLE
Explain how to manage Kubernetes running on HUAWEI CLOUD provider.

### PR DESCRIPTION
**Pull request by RainbowMango**
Add [kubernetes-sigs/cloud-provider-huaweicloud](https://github.com/kubernetes-sigs/cloud-provider-huaweicloud) documentation.

Fixes https://github.com/kubernetes-sigs/cloud-provider-huaweicloud/issues/69

### CODE REVIEW

**Comment by RainbowMango** [comment]
/assign @jimangel

@ reviewers, you can get the page [from here](https://deploy-preview-20579--kubernetes-io-master-staging.netlify.app/docs/concepts/cluster-administration/cloud-providers/#huawei-cloud).

**Comment by RainbowMango** [comment]
> who can provide technical review on this documentation change (to verify that it is accurate)?

@kevin-wangzefeng is one of the maintainers of `kubernetes-sigs/cloud-provider-huaweicloud`, maybe he can help to confirm.

File: content/en/docs/concepts/cluster-administration/cloud-providers.md [hunk 1]
```diff
@@ -134,6 +134,15 @@ If you wish to use the external cloud provider, its repository is [kubernetes/cl
 The GCE cloud provider uses the hostname of the node (as determined by the kubelet or overridden with `--hostname-override`) as the name of the Kubernetes Node object.
 Note that the first segment of the Kubernetes Node name must match the GCE instance name (e.g. a Node named `kubernetes-node-2.c.my-proj.internal` must correspond to an instance named `kubernetes-node-2`).
 
+## HUAWEI CLOUD
+
+If you wish to use the external cloud provider, its repository is [kubernetes-sigs/cloud-provider-huaweicloud](https://github.com/kubernetes-sigs/cloud-provider-huaweicloud).
+
+### Node Name
+
+The HUAWEI CLOUD provider uses the private IP address of the node (as determined by the kubelet or overridden with `--hostname-override`) as the name of the Kubernetes Node object.
```
Comment (kevin-wangzefeng) [inline content/en/docs/concepts/cluster-administration/cloud-providers.md:143]:
```suggestion
The HUAWEI CLOUD provider needs the private IP address of the node as the name of the Kubernetes Node object.
Please make sure indicating `--hostname-override=<node private IP>` when starting kubelet on the node.
```

File: content/en/docs/concepts/cluster-administration/cloud-providers.md [hunk 2]
```diff
@@ -134,6 +134,16 @@ If you wish to use the external cloud provider, its repository is [kubernetes/cl
...
 
+## HUAWEI CLOUD
+
+If you wish to use the external cloud provider, its repository is [kubernetes-sigs/cloud-provider-huaweicloud](https://github.com/kubernetes-sigs/cloud-provider-huaweicloud).
+
+### Node Name
+
+The HUAWEI CLOUD provider needs the private IP address of the node as the name of the Kubernetes Node object.
+Please make sure indicating `--hostname-override=<node private IP>` when starting kubelet on the node.
+Note that the Kubernetes Node name must match the HUAWEI CLOUD VM private IP.
```
Comment (kevin-wangzefeng) [inline content/en/docs/concepts/cluster-administration/cloud-providers.md:145]:
```suggestion
```

**Pull request merged**